### DISCUSSION TITLE
Use templates and thumbnails imported as pip module as default value as email setting

### DISCUSSION THREAD

**Discussion by Corinari** [pretix/pretix#2093] [Q&A]
Hello
I've written an email-plugin and wrapped it in a Pip package for an easy install. Now I am required to include the custom HTML Template for the email and the custom thumbnail in the pip package.
As far as i know, you can only choose to use templates located in the base/templates/base/email folder (which requires me to copy my templates and thumbnail to this location after pip installation)
Is there any other way to achieve this?

Sorry if this question seems easy/trivial, I have no experience regarding pip or Django prior this.

Thanks for any help in advance

**Comment by raphaelm** [comment]
There's no problem adding custom HTML templates or images to your own plugins, you can just look at the different plugins at https://github.com/pretix for examples
